mammoth/style_reader/document_matcher_reader.py

```diff
@@ -90,6 +92,10 @@ def _read_list_node(list_node):
 list = ":" list_type "(" ~"[0-9]+" ")"
 
 list_type = "ordered-list" / "unordered-list"
+
+line_break = "br" line_break_type?
+
+line_break_type = "page" / "column"
```

Comment:
As a simple example, do you understand how the style mappings for underlines work? The relevant pieces would be the document matcher itself (in `document_matchers.py`), parsing the document matcher (in `document_matcher_reader.py`) and then using the document matcher (in `conversion.py`).